usaspending_api/search/v2/views/spending_by_award_count.py

```diff
@@ -71,6 +72,7 @@ def post(self, request):
             results = self.handle_subawards(filters)
         else:
             results = self.handle_awards(filters, empty_results)
+        results["message"] = get_time_period_message()
```

Comment:
Should the message be nested under `results`?

Reply:
Good catch.  Should be fixed.